mllib/src/main/scala/org/apache/spark/ml/feature/StopWordsRemover.scala

```diff
@@ -84,7 +86,36 @@ class StopWordsRemover @Since("1.5.0") (@Since("1.5.0") override val uid: String
   @Since("1.5.0")
   def getCaseSensitive: Boolean = $(caseSensitive)
 
-  setDefault(stopWords -> StopWordsRemover.loadDefaultStopWords("english"), caseSensitive -> false)
+  /**
+   * [[https://docs.oracle.com/javase/8/docs/api/java/util/Locale.html Locale]] of the input.
+   * Ignored when [[caseSensitive]] is false.
+   * Default: Locale.English
+   * @see `StopWordsRemover.loadDefaultStopWords()`
+   * @group param
+   */
+  @Since("2.4.0")
+  val locale: Param[Locale] = new Param[Locale](this, "locale",
```

Comment:
Please use `Param[String]` so we can add Python/R APIs easily.

Reply:
+1